internal/commands/server/k8s_e2e_test.go

```diff
@@ -1,5 +1,4 @@
 //go:build e2e
-// +build e2e
```

Comment:
This is the old format which was replaced by the `go:build` format in go 1.17 ([docs](https://pkg.go.dev/cmd/go#hdr-Build_constraints))